apps/condo/domains/document/hooks/useUpdateDocumentModal.tsx

```diff
@@ -0,0 +1,172 @@
+import { Document as DocumentType } from '@app/condo/schema'
+import { Form, notification, Space } from 'antd'
+import dayjs from 'dayjs'
+import get from 'lodash/get'
+import React, { CSSProperties, useCallback, useMemo, useState } from 'react'
+
+import { Download, Paperclip } from '@open-condo/icons'
+import { useIntl } from '@open-condo/next/intl'
+import { Button, Modal, Typography } from '@open-condo/ui'
+import { colors } from '@open-condo/ui/dist/colors'
+
+import { FormWithAction } from '@condo/domains/common/components/containers/FormList'
+import { useDownloadFileFromServer } from '@condo/domains/common/hooks/useDownloadFileFromServer'
+import { DocumentCategoryFormItem } from '@condo/domains/document/components/DocumentCategoryFormItem'
+import { Document } from '@condo/domains/document/utils/clientSchema'
+
+
+const FILE_WRAPPER_STYLE: CSSProperties = { width: '100%', backgroundColor: colors.gray[1], borderRadius: '8px', padding: '16px' }
```

Comment:
I thought we've moved to css modules? They aren't supported in condo yet?

Reply:
No, I tried but they don’t want to work correctly) 
We should to research that